.github/actions/rust-prepare/action.yml

```diff
@@ -5,18 +5,13 @@ inputs:
     description: "SSH socket"
     default: "/tmp/ssh_agent.sock"
     required: true
-  gitlab_ssh:
-    description: "Gitlab ssh"
-    required: true
-  gitlab_user:
-    description: "Gitlab username"
-    required: true
-  gitlab_pass:
-    description: "Gitlab username"
-    required: true
   cargo_home:
     description: "Cargo home"
     default: ".cargo"
+    required: true
```

Comment:
why cargo_home required?

Reply:
`prepare_rust.sh` creates and puts stuff in the cargo home directory. Most
likely there was a good reason to allow a different name for it but I do not
know it.

Reply:
yeah but the default value applies if not set, why does it need to be required? ^^

The reason it exists is btw mostly gitlab ci / our old runners I think

Reply:
ah thanks sry that was a mistake